script/DeployLocalDevNet.s.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-FileCopyrightText: 2024 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.24;
+
+import { DeployBase } from "./DeployBase.s.sol";
+import { GIndicies } from "./constants/GIndicies.sol";
+
+contract DeployLocalDevNet is DeployBase {
+    constructor() DeployBase("local-devnet", 32382) {
+        // Lido addresses
+        config.lidoLocatorAddress = vm.envAddress("CSM_LOCATOR_ADDRESS");
```

Comment:
Should we add a custom wrapper for the vm.envAddress so that it reverts if the var is not set?

Reply:
it should fail, for the opposite there is `vm.envOr` with some default value